packages/manager/apps/dedicated/client/app/ip/components/list/firewall/game/ip-ip-firewall-game.controller.js

```diff
@@ -5,8 +5,10 @@ import startCase from 'lodash/startCase';
 import { IP_MITIGATION_RULE_PROTOCOL_PORT } from './ip-ip-firewall-game.constants';
 
 export default /* @ngInject */ function IpGameFirewallCtrl(
+  $http,
   $location,
   $scope,
+  $stateParams,
```

Comment:
**nitpick:** Can you please send the stateParam as an input and avoid injecting `$stateParams` service into the controller.